CRM/Member/Form/Membership.php

```diff
@@ -1344,8 +1344,6 @@ public function submit(): void {
       if ($paymentStatus !== 'Completed') {
         $params['status_id'] = $pendingMembershipStatusId;
         $params['skipStatusCal'] = TRUE;
-        // unset send-receipt option, since receipt will be sent when ipn is received.
-        unset($formValues['send_receipt'], $formValues['send_receipt']);
```

Comment:
keep the integrity of $formValues - look up the paymentStatus further down to keep the behavious